api/v2/checluster_types.go

```diff
@@ -862,6 +879,18 @@ func (c *CheCluster) GetIdentityToken() string {
 	return constants.IdToken
 }
 
+// Returns the default container security context required for container builds.
+// Returns an error if the default container security context could not be retrieved.
+func (c *CheCluster) GetDefaultContainerSecurityContext() (*corev1.SecurityContext, error) {
```

Comment:
Could you move this func to `dev_workspace_config.go` ?

Reply:
Will do 👍